ang/civiawards.ang.php

```diff
@@ -8,6 +8,12 @@
  * http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_angularModules.
  */
 
+$options = [
+  'awardTypes' => 'civiawards_award_type',
+];
+
+set_option_values_to_js_vars($options);
```

Comment:
Not sure if Civicrm will run the function in the parent extension first. Civi might and it might not :(. To be on the safe side, I suggest we have a `set_option_values_to_js_vars` function on this file or better still if this will end up being used by Civicase, CiviAwards, Prospects and any other Civicase extension, I suggest we extract the logic to a class in Civicase and use it anywhere we need it.
@deb1990 , @reneolivo